src/views/CampusSafety/views/LostAndFound/index.tsx

```diff
@@ -95,6 +108,19 @@ const LostAndFound = () => {
   const handleModalSubmit = async () => {
     try {
       await lostAndFoundService.updateReportStatus(parseInt(reportToDelete || ''), 'deleted');
+      let actionRequestData = {
+        ...newActionFormData,
+        missingID: reportToDelete,
```

Comment:
```suggestion
        missingID: parseInt(reportToDelete || ''),
```
This will fix the Typescript error on line 122.